### DISCUSSION TITLE
Support markdown / basic formatting / links in PL Network Directory

### DISCUSSION THREAD

**Discussion by bmann** [memser-spaceport/pl-network-hub#18] [Ideas]
I just updated my profile and the one for Fission in the directory -- very nice to be able to do this myself, thank you!

A feature request I have is to support markdown or at least basic formatting and links in the long description for a project/company

<img width="615" alt="Screen Shot 2023-07-18 at 9 56 18 AM" src="https://github.com/memser-spaceport/pl-network-hub/assets/280420/206bb2e7-8d45-4d56-9a2e-cf0a731fe8e0">

In this example, I'd like to link ODD and maybe links for the protocols, too.

**Comment by pandeyanujk** [comment]
Hey @bmann This is a good feature request,thanks for the feedback!
